packages/builder-rsbuild/package.json

```diff
@@ -85,12 +85,12 @@
     "@types/pretty-hrtime": "^1.0.3",
     "pretty-hrtime": "^1.0.3",
     "slash": "^5.1.0",
-    "storybook": "8.4.0-alpha.0",
+    "storybook": "8.4.0-alpha.4",
     "typescript": "^5.6.2"
   },
   "peerDependencies": {
     "@rsbuild/core": "^1.0.1",
-    "storybook": "^8.2.1"
+    "storybook": ">= 8.4.0-alpha.2 || >= 8.4.0"
```

Comment:
cc @JReinhold, may I know will the change will be back port to 8.3.x or 8.2.x to make the peer deps compatible with more version?

Reply:
We're not backporting, but the changes you're introducing here shouldn't break usage of Storybook <8.4. you're just removing usage of the handlebars export, that will work on 8.0 etc. too

Reply:
Oh, that's right. I'll revert the change here, thanks for correcting.